django/src/rdwatch/views/site_image.py

```diff
@@ -146,3 +151,71 @@ def site_images(request: HttpRequest, id: UUID4):
     output['evaluationGeoJSON'] = site_eval_data['json']['evaluationGeoJSON']
     output['evaluationBBox'] = site_eval_data['json']['evaluationBBox']
     return output
+
+
+@router.post('/{id}/image_embedding', response=UUID4)
+def post_image_embedding(request: HttpRequest, id: int):
+    if not SiteImage.objects.filter(pk=id).exists():
+        raise Http404()
```

Comment:
This can be done more concisely with 
```suggestion
	get_object_or_404(SiteImage, pk=id)
```